pkg/pow/miningTasker.go

```diff
@@ -132,12 +137,32 @@ func (mt *MiningTasker) GetWork(chan *Work) (*Work, bool) {
 		m2, err := mt.proxy.BatchGet([]string{valKey})
 		if err != nil {
 			mt.log.Info("Could not retrieve pricing data for current request id: %v", err)
-			return nil, false
+			//return nil, false
 		}
 		val := m2[valKey]
 		if len(val) == 0 {
-			mt.log.Info("Pricing data not available for request %d", reqIDs[i].Uint64())
-			return nil, false
+			cfg := config.GetConfig()
+			indexPath := filepath.Join(cfg.ConfigFolder, "manualData.json")
+			jsonFile, err := os.Open(indexPath)
+			if err != nil {
+				fmt.Println("manualData read error", err)
+				return nil, false
+			}
+			defer jsonFile.Close()
+			byteValue, _ := ioutil.ReadAll(jsonFile)
+			var result map[string]map[string]uint
+			err = json.Unmarshal([]byte(byteValue), &result)
+			if err != nil {
+				fmt.Println("manualData unsmarshal ", err)
```

Comment:
Lets not use Println anymore :) there is already a logger here.
```suggestion
				mt.log.Error("manualData unsmarshal err:%v", err)
```